### PR TITLE
Multiple images issue180

### PR DESCRIPTION
**Pull request by sachintha-s**
Extended the chat input widget to send multiple images as per [issue180.](https://github.com/effektio/effektio/issues/180)
image

### CODE REVIEW

File: app/lib/common/widget/custom_chat_input.dart [hunk 1]
```diff
@@ -7,10 +7,20 @@ import 'package:get/get.dart';
...
-  CustomChatInput({Key? key, required this.context}) : super(key: key);
+  CustomChatInput({
+    Key? key,
+    required this.context,
+    required this.isChatScreen,
+    required this.onButtonPressed,
+    required this.roomName,
+  }) : super(key: key);
   final BuildContext context;
+  final Function()? onButtonPressed;
```
Comment (bitfriend) [inline app/lib/common/widget/custom_chat_input.dart:20]: If this is the required field, `?` may be not necessary.

File: app/lib/screens/ChatProfileScreen/ImageSelectionScreen.dart [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+import 'package:photo_view/photo_view.dart';
+import 'package:photo_view/photo_view_gallery.dart';
+
+class ImageSelection extends StatefulWidget {
+  const ImageSelection({
+    Key? key,
+    required this.imageList,
+    required this.roomName,
+  }) : super(key: key);
+  final List? imageList;
```
Comment (bitfriend) [inline app/lib/screens/ChatProfileScreen/ImageSelectionScreen.dart:18]: If this is the required field, `?` may be not necessary.

File: app/lib/common/widget/custom_chat_input.dart (hunk 1 above)
Comment (gtalha07) [reply app/lib/common/widget/custom_chat_input.dart:20]: Agree.

**Review by bitfriend — APPROVED** [review]
Great work.
And I left some comments.
You can fix them prior to merge.

**Review by gtalha07 — APPROVED** [review]
LGTM.

**Pull request merged**